db/migrate/20200303222558_add_context_to_rating_votes.rb

```diff
@@ -0,0 +1,5 @@
+class AddContextToRatingVotes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :rating_votes, :context, :string, default: "explicit"
```

Comment:
You might want to add a unique index as the model is defined like this now:

```ruby
validates :user_id, uniqueness: { scope: %i[article_id context] }
```

something like this I guess

```ruby
add_index :rating_votes, [:user_id, :article_id, :context], unique: true
```

Reply:
✅